### DISCUSSION TITLE
Capytaine apply to OSWC?

### DISCUSSION THREAD

**Discussion by saltynexus** [capytaine/capytaine#54] [General]
I've been looking at testing NEMOH in an ongoing OSWC project, but it seems Capytaine is more user friendly (code, docs, tutorials, etc). Furthermore, it seems NEMOH has lost steam in its development (specifically with engaging the user community), so I like what I see here so far. I'm wondering if Capytaine can be applied to an OSWC without major development? Any feedback (even if just general overview) would be helpful. Depending on things evolve, I might even be able to help contribute a test case / tutorial with such an application.

**Comment by mancellin** [comment]
I don't have an example of application of Capytaine to an OWSC, but I don't expect any major issue. According to [this paper](http://www.eeng.nuim.ie/jringwood/Respubs/C283MPEW.pdf) Nemoh is reasonably accurate for this application and the same should hold for Capytaine.

**Comment by saltynexus** [comment]
I have read that paper and it was one of the motivations for trying NEMOH. Thanks for indicating that you foreseen no issues in applying Capytaine...I think I'm going to give it a try.
